src/main/webapp/admin/storeAccessory.js

```diff
@@ -0,0 +1,32 @@
+const filepathRoot = "../images/accessories/";
+
+/** Show uploaded accessory image */
+function getImage(filepathStub) {
+  const accessoryPicture = document.getElementById('accessory');
+  accessoryPicture.src = filepathRoot + filepathStub;
+  accessoryPicture.onload = function() {
+    URL.revokeObjectURL(accessoryPicture.src) // free memory
+  }
+  accessoryPicture.style.zIndex = "3"; // on top of player face and body
+  accessoryPicture.style.position = "absolute"; // on top of player face and body
+}
+
+function changeAccessory(name, value) {
+  value = value.toString() + "px";
+  const accessoryPicture = document.getElementById('accessory');
+  switch(name) {
+  case "height":
+    accessoryPicture.style.height = value;
+    break;
+  case "width":
+    accessoryPicture.style.width = value;
+    break;
+  case "xPos":
+    accessoryPicture.style.left = value;
+    break;
+  case "yPos":
+    accessoryPicture.style.top = value;
+    break;
+  }
+}
+ 
```

Comment:
new line

Reply:
Hmm I think there is a new line, just not showing up in the commit diff: https://github.com/googleinterns/step144-2020/blob/accessory-front/src/main/webapp/admin/storeAccessory.js

Reply:
I think its got 2 new lines? ether that or whitespace